### PR TITLE
Update cryptocurrencies.yml

### PR DESCRIPTION
**Pull request by deadr0ck**
- Updated formatting to current format (bcc/btc)
- Removed BTCjam
- Added Bithumb, BTER, CoinDance, CoinMarketCap, CoinMix, Changelly, Evercoin, Korbit, QuadrigaCX, Shapeshift.

### CODE REVIEW

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -2,231 +2,336 @@ websites:
...
-      sms: Yes
-      software: Yes
-      doc: https://cex.io/guide#4
+      bcc: Yes
+      btc: Yes
+      othercrypto: Yes
+      doc: https://blog.cex.io/news/bitcoin-cash-deposits-withdrawals-enabled-cex-io-16432
+
+    - name: Changally
+      url: https://shapeshift.io
```
Comment (rawb0t) [inline _data/cryptocurrencies.yml]: is changelly shapeshift?

Comment (kenman345) [reply _data/cryptocurrencies.yml]: they're similar but competing products I believe. So yea, that needs fixing too

Comment (t0p0lin0) [reply _data/cryptocurrencies.yml]: I don't believe Changelly is Shapeshift. They're similar, but I am pretty sure they're separate entities.

**Comment by deadr0ck** [comment]
Fixed as requested by @rawb0t

**Comment by rawb0t** [comment]
Fixes look good. Just need icons for the additions now

**Comment by deadr0ck** [comment]
Still many more to add to this category. Should we consider breaking "cryptocurrencies" out to multiple categories? Consider that many of the listed sites will be related to crypto.

Can I suggest splitting into (at least) these categories: 1) Exchanges 2) Wallets 3) Services

Thoughts or suggestions?

**Comment by deadr0ck** [comment]
Services would be strictly crypto-related items that don't quite fit under Exchange, Wallet, or any others. Maybe call it 'Crypto Services' ?

Ie. Coin mixers, Block Explorers, Mining operators

**Comment by rawb0t** [comment]
name of new services cat probably needs some discussion but that can be done elsewhere than this PR.

i think you've covered all of the changes i requested.

**Pull request merged**